UnleashSampleApp/ViewController.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  ViewController.swift
+//  UnleashSampleApp
+//
+//  Copyright © 2019 Silvercar. All rights reserved.
+//
+
+import UIKit
+import Unleash
+
+class ViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let unleash = Unleash(appName: "rac-web-admin-dev",
+                              url: "https://unleash.silvercar.com/api",
+                              refreshInterval: 300000,
+                              strategies: [EnvironmentStrategy()])
+        print("hello")
```

Comment:
That shouldn't be there, I'm going to wait on the other reviews and then do a force push removing this.